README.md

```diff
@@ -1,27 +1,42 @@
-# James Webb Space Telescope Data Analysis Tool Notebooks
+James Webb Space Telescope Data Analysis Tool Notebooks
+--------------
```

Comment:
```suggestion
# James Webb Space Telescope Data Analysis Tool Notebooks
```
Why this change? the `# XXX` is markdown style for headers, and since this file's name is `.md`, it should follow markdown format.

Or did you mean to produce a horizontal line?  If so, you probably want this:
```suggestion
# James Webb Space Telescope Data Analysis Tool Notebooks
--------------
```
Since you still want it to be in header style